test/e2e/helm/test_kserve_sklearn.py

```diff
@@ -64,11 +64,13 @@ def test_sklearn_kserve():
     kserve_client = KServeClient(
         config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
     kserve_client.create(isvc)
-    kserve_client.wait_isvc_ready(
-        service_name, namespace=KSERVE_TEST_NAMESPACE)
-
-    res = predict(service_name, "./data/iris_input_v2.json",
-                  protocol_version="v2")
-    assert res["outputs"][0]["data"] == [1, 1]
-
-    kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
+    # TODO: Temporarily disabled. Will need to investigate a new
```

Comment:
I think the storage_uri might need to fix as well. @sivanantha321 might have better idea

Reply:
@sivanantha321 Could you point me to where those tests are?

Reply:
https://github.com/search?q=repo%3Akserve%2Fkserve+%22gs%3A%2F%2Fseldon-models%2Fsklearn%2Fmms%2Flr_model%22&type=code

Reply:
@sivanantha321 I am seeing this error after updating the URI. Any ideas?
```
FAILED predictor/test_sklearn.py::test_sklearn_v2_grpc - assert [] == [1, 1]
  Right contains 2 more items, first extra item: 1
  Full diff:
  - [1, 1]
  + []
```

Reply:
@terrytangyuan The output is returned as int_contents instead of int64_contents.